### PR TITLE
Making Website Responsive for Desktop

### PR DESCRIPTION
**Pull request by mrEckendonk**
## Hi :hand:  Code Reviewer 

For this exercise, I used media Querries to re-organize the layout of the already mobile version to desktop / bigger screen.
We didn't have to use JavaScript, but the template has 6 images where the button should be disabled, and when hovering over the image the button should appear again with another image. DOM manipulation is not used in this exercise, nor taught in the Microverse program.

- [x] media Querries are set at 996px
- [x] Used the Figma template 2 layout

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -574,8 +635,243 @@ button:active {
...
+  .hr-line-bottom {
+    width: 100%;
+  }
+
+  .social-icons-bottom {
+    display: flex;
+    margin-left: 50%;
+    margin-right: 50%;
+    width: 50%;
+  }
```
Comment (D4nielJ) [inline style.css:908]: - [x] Let's try to fix this footer, right now is overflowing to the right of the page creating blank space all over your app on the right.

image

File: style.css [hunk 2]
```diff
@@ -574,8 +635,243 @@ button:active {
...
+  .container-section2 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .backgroundabout-me {
+    background: url("assets/background_desktop.svg");
+    background-repeat: no-repeat;
+    background-position: right;
+  }
```
Comment (D4nielJ) [inline style.css:805]: - [x] In the mobile view, it's the paragraph in this section then one causing the page to overflow. Let's try to fix it aswell.

image

File: index.html [hunk 3]
```diff
@@ -197,9 +209,38 @@ <h2>Professional Art Printing Data</h2>
         >
           See Project
         </button>
-      </article>
-      <article class="professional-art">
-        <h2>Professional Art Printing Data</h2>
+      </div>
+      </div>
+      <div class="container-section2">
```
Comment (D4nielJ) [inline index.html:214]: [OPTIONAL] In the desktop view, some of the elements are not correctly centered across the app. We could try to fix these by taking advantage of the flex properties like `justify-content` and `align-items`. 

My guess is that you're using a small screen to develop your app, and while this is not bad at all, designing for every screen is important. My advice in this case is to take advantage of the devtools of the web browser you're using to not only make the screen smaller, but also bigger and emulate bigger screens. [You can take a look at this ranking of common screen sizes to know what to develop for.](https://www.google.com/search?q=rank+of+screen+sizes+desktop&oq=rank+of+screen+sizes+desktop&aqs=chrome..69i57j33i22i29i30.5847j0j1&sourceid=chrome&ie=UTF-8)


image

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @Eckybrazzz, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (D4nielJ) [reply style.css:908]: Next reviewer: It appears that my web browser bugged while I was inspecting the page. This error doesn't appear to be present.

File: style.css (hunk 2 above)
Comment (mrEckendonk) [reply style.css:805]: Fixed by setting a width of 327px to element

File: index.html (hunk 3 above)
Comment (mrEckendonk) [reply index.html:214]: Also fixed. Now it looks even better.

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @Eckybrazzz,  😊

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost there

 ## Required Changes ♻️

- [x] Seems that `-webkit-box-sizing: inherit;` is causing one of the major problems. Let me show you.

- Here is with `-webkit-box-sizing: inherit;`

Screen Shot 2021-11-04 at 18 45 52

- Here is without  `-webkit-box-sizing: inherit;`

Screen Shot 2021-11-04 at 18 45 57

- [x] Also in the class 'form-text' the `width: 648px;` of the textarea is causing problems.
* don't put attention to the color, is just to see more clearly the bug.

Screen Shot 2021-11-04 at 19 00 38

- [x]  Without the width: 648px.

Screen Shot 2021-11-04 at 19 01 13

- [x] Kindly fix those bugs as the last code reviewer suggested it would improve your app in a huge measure. 


_Check the comments under the review._

## Optional suggestions

- [x] Try to stick with Figma design as much as possible. Cheers 😊

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Eckybrazzz** [comment]
@eri8-9 I have sent you a message on Slack. But noticed that you don't have it in your template as a way to communicate.
Can you quickly check the corrections before I ask for a new review as it is the last chance that I have?

Thanks :wave:

**Comment by D4nielJ** [comment]
Hi @Eckybrazzz, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that do not include `node_modules` in the `.gitignore` file. Please do not commit files that are not related to the project, since they make your repository look unprofessional and your pull requests are harder to review.

[You can read more about how to keep your repository professional here.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 4]
```diff
@@ -13,18 +13,32 @@
...
+                <a class="desktop-link" href="#">Contact</a>
+              </li>
+              <li>
+                <a href="#"><img src="assets/icon-mail.svg" alt="mail" /></a>
+              </li>
+            </ul>
+          </nav>
       </div>
     </header>
     <section class="intro background background-desktop">
```
Comment (alonsofalconi) [inline index.html:44]: - [OPTIONAL] You have done a great job addressing the changes requested by the previous reviewers. But it seems that there is a new issue after the new changes. Your headline has more space than it has before, I think it won't take you too much time to fix it. I strongly suggest you fix this issue before moving to the next milestone :wink: :raised_hands: 

  Your design now:
  image

  Your design before:
  image

File: index.html [hunk 5]
```diff
@@ -251,50 +291,42 @@ <h2>Professional Art Printing Data</h2>
...
+      </div>
+    </div>
+    
     <!-- Start About me-->
-    <section class="about-me">
-      <article>
+    <div class="about-me">
+      <div class="backgroundabout-me">
       <div class="first">
         <h2>About me</h2>
```
Comment (alonsofalconi) [inline index.html:302]: - [ ] [OPTIONAL] I think your portfolio has a few alignment issues that have not been addressed before and I will mark these as [OPTIONAL] but I strongly suggest working on these before moving to the next milestone :pushpin: :raised_hands:

  About section: The "About me" title should be aligned with the paragraph :pushpin:
  image

  Knowledge section: I should be centered according to the Figma design :wink:
  image

  Contact form section: This form should also be centered in order to match the Figma design :hugs:
  image

**Review by alonsofalconi — APPROVED** [review]
Hi @Eckybrazzz,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -17,19 +17,32 @@
...
+            </li>
+            <li>
+              <a class="desktop-link" href="#">Contact</a>
+            </li>
+            <li>
+              <a href="#"><img src="assets/icon-mail.svg" alt="mail" /></a>
+            </li>
+          </ul>
         </nav>
-
```
Comment (vabuyia) [inline index.html:27]: #For code clarity , such should be removed.

File: index.html [hunk 7]
```diff
@@ -17,19 +17,32 @@
     <header>
       <div class="top-bar">
         <div class="top-bar-left">
-          <a href="#"
-            ><span class="top-bar-left-name">Mike</span></a
-          >
+          <a href="#"><span class="top-bar-left-name">Mike</span></a>
         </div>
         <nav class="top-bar-icon">
-          <a href="#"><img src="assets/Union.svg" alt="Menu Mobile" /></a>
```
Comment (vabuyia) [inline index.html:25]: It is better to link a tag so that it can be clickable.

**Review by vabuyia — COMMENTED** [review]
So far so good, kindly make changes from the below comments.

File: index.html (hunk 4 above)
Comment (mrEckendonk) [reply index.html:44]: I removed as suggested by the previous code reviewer the `-webkit-box-sizing: inherit;` and that changed the top nav bar and the layout of some parts. I will correct the location of the text

**Pull request merged**